MediaBrowser.Channels.LeagueOfLegends/LeagueOfLegendsChannel.cs

```diff
@@ -133,23 +134,23 @@ public ChannelParentalRating ParentalRating
             get { return ChannelParentalRating.GeneralAudience; }
         }
 
-        public async Task<IEnumerable<ChannelMediaInfo>> GetChannelItemMediaInfo(string id, CancellationToken cancellationToken)
+        public async Task<IEnumerable<MediaSourceInfo>> GetChannelItemMediaInfo(string id, CancellationToken cancellationToken)
         {
             if (Helpers.IsPlaceholderId(id))
             {
                 // Default videos when the game doesn't actually exists
-                return await RetrieveRickAstleyChannelMediaInfo(cancellationToken);
+                return await RetrieveRickAstleyMediaSourceInfo(cancellationToken);
```

Comment:
I have no clue. Also the changed line is a part of a commit that I think I should just nuke altogether because it didn't do what I hoped. What do you think? What should the ChannelMediaInfo stuff be ported to? It seems to collide with the server a lot..

Reply:
I reverted it